src/content/datachannel/datatransfer/js/main.js

```diff
@@ -24,6 +24,13 @@ var bytesToSend = 0;
 
 sendButton.onclick = createConnection;
 
+// Prevent data sent to be set to 0.
+megsToSend.addEventListener('change', function(e) {
+  if (this.value === 0) {
+    megsToSend.value = 1;
```

Comment:
This works, though it's not explicit (if someone tries out zero to see what happens). I think it would be better to disallow values of zero or less and display a status message that says that only positive values are allowed.

Also noticed that send channel state is never closed if zero is input, or a negative value.

(Frustrating that it's only possible to set the min spinner value or validate the input value, and not  possible to disallow values less than or equal to zero without extra code.)